src/containerapp/azext_containerapp/_utils.py

```diff
@@ -81,8 +81,9 @@ def process_service(cmd, resource_list, service_name, arg_dict, subscription_id,
                 service_bind = {
                     "serviceId": containerapp_def["id"],
                     "name": binding_name,
-                    "clientType": arg_dict.get("clientType")
                 }
+                if arg_dict.get("clientType"):
```

Comment:
then how can customer clear clilent type?

Reply:
If the Cx want to clean the clientType, input `clientType=none`